.github/workflows/osx_static_cmake.yml

```diff
@@ -60,8 +60,7 @@ jobs:
         uses: ./.github/actions/install-deps-osx
       - name: test
         env:
```

Comment:
Please remove this, otherwise, you will get a syntax error.

Reply:
Shit. Thanks for noticing! Fixed.